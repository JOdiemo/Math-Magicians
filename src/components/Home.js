import React from 'react';

const Home = () => (
  <div className="SectionContainer">
    <div className="ChalkBoard SectionContentContainer">
      <div className="SectionContent">
        <h1>Math Magicians</h1>
        <h2>Welcome to our page</h2>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          Duis id turpis at urna convallis laoreet.Ut malesuada,
          urna ac dignissim maximus, sapien mauris mollis enim,
          et convallis ex tellus a erat. Nulla suscipit erat sed
          mauris placerat, nec aliquet tortor maximus. Sed quis turpis
          ut lacus commodo pharetra ac at ante. Vivamus ac sollicitudin lectus.
          Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere

          cubilia curae; Donec non lorem aliquam, scelerisque lorem sit amet, interdum odio.
          Maecenas risus ipsum, molestie a augue nec, auctor iaculis leo. Proin semper
          fringilla dui, feugiat rhoncus sapien volutpat lacinia. Aliquam cursus sem non
          tincidunt imperdiet. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          Quisque dapibus et enim nec consequat. Quisque imperdiet tempor faucibus.
          {' '}
        </p>
      </div>
    </div>
  </div>
);

export default Home;
