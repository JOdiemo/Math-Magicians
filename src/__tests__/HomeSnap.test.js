import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

describe('Snapshot of home page', () => {
  it('Should give smilar Homepage shot', () => {
    const HomeSnap = renderer.create(<Home />).toJSON;
    expect(HomeSnap).toMatchSnapshot();
  });
});
