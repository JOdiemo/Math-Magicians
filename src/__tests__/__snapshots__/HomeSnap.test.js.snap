// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of home page Should give smilar Homepage shot 1`] = `[Function]`;
